src/event.rs

```diff
@@ -389,6 +389,8 @@ pub enum WindowEvent {
     /// Applications might wish to react to this to change the theme of the content of the window
     /// when the system changes the window theme.
     ///
+    /// This only reports a change if the window theme was not overridden by the application.
+    ///
```

Comment:
I think the previous sentence says the same thing. `system changes the window theme` is not a user changes the theme with `Window::set_theme`.

Reply:
I've changed this to reference `Window::set_theme` specifically instead